recipes/slider/demos/slider-demo/index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <title>DEMO — Слайдер — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    *,
+    *::before,
+    *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    html {
+      color-scheme: dark;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: #18191C;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+    }
+
+    h3 {
+      margin: 37px auto 0;
+      font-size: 24px;
+      line-height: 26px;
+
+    }
+
+    @media (max-width: 768px) {
+      body {
+        padding: 30px;
+      }
+    }
+
+    .button_type_radio {
+      background-color: transparent;
+      margin: 0;
+      padding: 0;
+      width: 15px;
+      height: 15px;
+      border-radius: 50%;
+      border: 1px solid #868A8F;
+      margin-right: 12px;
+    }
+
+    .button_type_radio.active {
+      background-color: #53d67b;
+    }
```

Comment:
```suggestion
    .button_type_radio {
      background-color: transparent;
      margin: 0;
      padding: 0;
      width: 15px;
      height: 15px;
      border-radius: 50%;
      border: 1px solid #FFFFFF;
      margin-right: 12px;
    }

    .button_type_radio.active {
      background-color: #C56FFF;
    }
```

Подставляю дока-цвета :)